test/e2e/framework/helpers/yunikorn/rest_api_utils.go

```diff
@@ -128,19 +128,14 @@ func (c *RClient) GetApps(partition string, queueName string) ([]map[string]inte
 	return apps, err
 }
 
-func (c *RClient) GetAppInfo(partition string, queueName string, appID string) (map[string]interface{}, error) {
-	apps, err := c.GetApps(partition, queueName)
+func (c *RClient) GetAppInfo(partition string, queueName string, appID string) (*dao.ApplicationDAOInfo, error) {
```

Comment:
This signature change should trigger updates in all callers. Looking at the code there are at least 7 callers of this function.
We also need to become consistent in the way we call and process: either always return a `dao` type object for the thing we request or the generic `[]interface{}`.  I think `dao` is the only correct answer.
We already use `dao` for queue and partition but not for application...